packages/fx-core/src/plugins/resource/bot/v2/constants.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ServiceType } from "../../../../common/azure-hosting/interfaces";
+
+export const languageMapping: { [key: string]: string } = {
```

Comment:
Good to see the refactor here. I am curious that why L7-L9, the key is not with "", but L13-L14 has "". Maybe we can build a class for this mapping and provide method easy to use. Let's sync tomorrow